namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/elasticache.tf

```diff
@@ -0,0 +1,37 @@
+################################################################################
+# Interventions Elasticache for ReDiS
+################################################################################
+
+module "hmpps_interventions_elasticache_redis" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=4.2"
+  cluster_name           = var.cluster_name
+  cluster_state_bucket   = var.cluster_state_bucket
+  application            = var.application
+  environment-name       = var.environment
+  is-production          = var.is_production
+  infrastructure-support = var.infrastructure_support
+  team_name              = var.team_name
+  number_cache_clusters  = var.number_cache_clusters
+  node_type              = "cache.t2.small"
+  engine_version         = "6.x"
+  parameter_group_name   = "default.redis6.x"
+  namespace              = var.namespace
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "hmpps_interventions_elasticache_redis" {
+  metadata {
+    name      = "hmpps-interventions-elasticache-redis"
```

Comment:
Is `elasticache-redis` sufficient? The secret will be in the namespace of `hmpps-interventions-dev`

Reply:
sounds good